jms/1.7/modules/ROOT/pages/jms-publish-consume.adoc

```diff
@@ -96,15 +120,30 @@ with the same Correlation ID to be used to publish the request message.
 </flow>
 ----
 
-=== Message ID Pattern
+=== Configure the Message ID Pattern
+
+The *MESSAGE_ID* pattern implies that the operation after publishing a message to the destination queue, consumes a message using a selector that expects a message with a message id of the same value as the message id of the request message.
+
+In the following example, in the first flow you configure the *Publish consume* operation to publish a message to the `targetDestination` with the current `payload` as its body and the JMSReplyTo header configured to the known queue setting the *Reply To* field to `replyToDestination`. Also, the operation has configured the *Request-Reply Pattern* field to *MESSAGE_ID*. +
```

Comment:
Same comment as before, but that's just a suggestion. Maybe Kim told you to write it like this, so you can ignore me.